users/migrations/0003_alter_profile_dob.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2 on 2021-04-13 15:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0002_test_data'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='profile',
+            name='dob',
+            field=models.DateField(blank=True, null=True),
```

Comment:
MHO, this file entirely could be skipped,
please update one API, see this DIFF
https://pastebin.com/rZXEZFsk

You will have to solve conflict of commenting 
0002_test_data.py

Other wise you need methods in model to receive insert dob